lib/internals/dynamics.js

```diff
@@ -133,68 +134,66 @@ module.exports = (
   const parseAuthzRoles = (authzApiResponse) => {
     const { roles, roleMappings } = authzApiResponse
 
-    if (!roles || !roleMappings || !roles.length || !roleMappings.length) {
-      return null
-    }
-
-    const parsedMappings = {}
-
-    roleMappings.forEach(mapping => {
-      const [id, name] = mapping.split(':')
-
-      parsedMappings[id] = name
-    })
-
     const byStatus = {}
     const byRole = {}
     const byOrg = {}
     const flat = []
```

Comment:
"Flat" describes what has happened to it, not what it is. It also has a naming collision with common actions on arrays and strings, which adds to the confusion. This did slow me down before when I expected it to be an action. (Non-blocking review)

Reply:
Imo, it does describe what it is